### PR TITLE
[docs] Changelog v1.1.0

### PR DESCRIPTION
**Pull request by AndreyBelym**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,56 @@
 # Changelog
 
+## v1.1.0 (2019-2-28)
+
+### :gear: TypeScript 3 Support ([#3401](https://github.com/DevExpress/testcafe/issues/3401))
+
+TypeScript test files can now use new syntax features introduced in [TypeScript 3.0](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-0.html) and [TypeScript 3.3](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-3.html).
```
Comment (dirk-pieterse) [inline CHANGELOG.md]: "use new syntax features introduced" -> use **the** new syntax features introduced

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,56 @@
...
+
+### :gear: Enhanced TypeScript Definitions for Client Functions ([#3431](https://github.com/DevExpress/testcafe/pull/3431)) by [@vitalics](https://github.com/vitalics)
+
+The updated type definitions allow TypeScript compiler to determine client function's return value type. Static typing now warns you when you call wrong methods for the return value.
```
Comment (dirk-pieterse) [inline CHANGELOG.md]: "The updated type definitions allow TypeScript compiler" -> The updated type definitions allow **the** TypeScript compiler

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,56 @@
...
+
+```js
+const getFoo = ClientFunction(() => 42);
+const foo    = await getFoo();
+foo.match(/foo/);
+```
+
+Prior to v1.1.0, an error occurred during test execution:
```
Comment (dirk-pieterse) [inline CHANGELOG.md]: "Prior to v1.1.0" -> Before v1.1.0

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,56 @@
...
+ - Chrome 72.0.3626 / Windows 10.0.0
+ Fixture 1
+ √ Test 1
+ √ Test 2
+ ...
+ × Test N
+   1) TypeError: foo.match is not a function
+```
+
+With v1.1.0, the TypeScript compiler throws an error before test are started:
```
Comment (dirk-pieterse) [inline CHANGELOG.md]: "before test"? Do you mean 'a' test?

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,56 @@
...
+
+```text
+$ testcafe chrome tests.ts
+  ERROR Cannot prepare tests due to an error.
+  Error: TypeScript compilation failed.
+
+  tests.ts (4, 2): Property 'match' does not exist on type 'number'.
```
Comment (dirk-pieterse) [inline CHANGELOG.md]: "does not exist on" Why 'on'? Do you mean 'in'?

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,56 @@
...
+```
+
+### Bug Fixes
+
+* TestCafe no longer ignores test and fixture metadata filters specified in the configuration file ([#3443](https://github.com/DevExpress/testcafe/issues/3443)) by [@NanjoW](https://github.com/NanjoW)
+* TestCafe no longer resolves placeholders to `null` in video path patterns ([#3455](https://github.com/DevExpress/testcafe/issues/3455))
+* Fixed the `KeyboardEvent`'s `key` property emulation for Safari ([#3282](https://github.com/DevExpress/testcafe/issues/3282))
+* TestCafe can now capture element screenshots after a long page scrolling ([#3292](https://github.com/DevExpress/testcafe/issues/3292))
```
Comment (dirk-pieterse) [inline CHANGELOG.md]: a long page scrolling?

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,56 @@
...
+* The compilation time of TypeScript tests no longer degrades for a large number of files ([#3475](https://github.com/DevExpress/testcafe/issues/3475))
```
Comment (dirk-pieterse) [inline CHANGELOG.md]: Explain.

Comment (dirk-pieterse) [reply CHANGELOG.md]: compilation time... degrades?

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,5 +1,56 @@
...
+* Reach Router can now navigate correctly when tested with TestCafe ([testcafe-hammerhead/#1863](https://github.com/DevExpress/testcafe-hammerhead/issues/1863))
```
Comment (dirk-pieterse) [inline CHANGELOG.md]: "when tested with TestCafe" It isn't necessary to add 'with TestCafe'

File: docs/articles/blog/2019-2-28-testcafe-v1-1-0-released.md [hunk 9]
```diff
@@ -0,0 +1,61 @@
...
+
+## Enhancements
+
+### ⚙ TypeScript 3 Support ([#3401](https://github.com/DevExpress/testcafe/issues/3401))
+
+TypeScript test files can now use new syntax features introduced in [TypeScript 3.0](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-0.html) and [TypeScript 3.3](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-3.html).
+
+### ⚙ Enhanced TypeScript Definitions for Client Functions ([#3431](https://github.com/DevExpress/testcafe/pull/3431)) by [@vitalics](https://github.com/vitalics)
+
+The updated type definitions allow TypeScript compiler to determine client function's return value type. Static typing now warns you when you call wrong methods for the return value.
```
Comment (dirk-pieterse) [inline docs/articles/blog/2019-2-28-testcafe-v1-1-0-released.md]: "allow TypeScript compiler" Do you mean 'the' TypeScript compiler?

File: docs/articles/blog/2019-2-28-testcafe-v1-1-0-released.md [hunk 10]
```diff
@@ -0,0 +1,61 @@
...
+ - Chrome 72.0.3626 / Windows 10.0.0
+ Fixture 1
+ √ Test 1
+ √ Test 2
+ ...
+ × Test N
+   1) TypeError: foo.match is not a function
+```
+
+With v1.1.0, the TypeScript compiler throws an error before test are started:
```
Comment (dirk-pieterse) [inline docs/articles/blog/2019-2-28-testcafe-v1-1-0-released.md]: "before test are started"? do you mean before **tests** are started?

File: CHANGELOG.md (hunk 6 above)
Comment (VasilyStrelyaev) [reply CHANGELOG.md]: Will it be more correct to say *after the page is scrolled for a long time*?

*TestCafe can now capture element screenshots after the page is scrolled for a long time*

File: CHANGELOG.md (hunk 7 above)
Comment (VasilyStrelyaev) [reply CHANGELOG.md]: I guess it's slang for performance degradation. We can say: 

*TypeScript compilation is now faster for a large number of files*

Comment (dirk-pieterse) [reply CHANGELOG.md]: Reduced TypeScript compilation time for a large number of files?

File: CHANGELOG.md (hunk 6 above)
Comment (dirk-pieterse) [reply CHANGELOG.md]: and what does 'after the page is scrolled for a long time' imply? Can you explain this?

**Pull request merged**